docs/core/tools/dotnet-sln.md

```diff
@@ -35,42 +34,64 @@ dotnet new sln
 
 - **`SOLUTION_FILE`**
 
-  The solution file to use. If not specified, the command searches the current directory for one. If there are multiple solution files in the directory, one must be specified.
+  The solution file to use. The command searches the current directory for one if it's left unspecified, failing if there are multiple solution files.
 
 ## Options
 
 - **`-h|--help`**
 
-  Prints out a short help for the command.
+  Prints options for the command.
 
 ## Commands
 
+### `list`
+
+Lists all projects in a solution file.
+
+#### Synopsis
+
+```dotnetcli
+dotnet sln list [-h|--help]
+```
+  
+#### Arguments
+
+- **`SOLUTION_FILE`**
+
+  The solution file to use. The command searches the current directory for one if it's left unspecified, failing if there are multiple solution files.
+
+#### Options
+
+- **`-h|--help`**
+
+  Prints out options for the command.
+  
 ### `add`
 
-Adds a project or multiple projects to the solution file.
+Adds one or more projects to the solution file.
 
 #### Synopsis
 
 ```dotnetcli
-dotnet sln [<SOLUTION_FILE>] add [--in-root] [-s|--solution-folder] <PROJECT_PATH>
+dotnet sln [<SOLUTION_FILE>] add [--in-root] [-s|--solution-folder <SOLUTION_FOLDER>] <PROJECT_PATH> [<PROJECT_PATH>...]
```

Comment:
We don't usually show the argument in the synopsis. Instead, we add that where the option is shown.

```suggestion
dotnet sln [<SOLUTION_FILE>] add [--in-root] [-s|--solution-folder] <PROJECT_PATH> [<PROJECT_PATH>...]
```

Reply:
I just saw another PR where a customer was trying to add the same info. Do you two think that we should do that across the board? I closed the PR for now so we could discuss that.

Reply:
If switches are well-named (as in `--solution-folder`), I don't think it's necessary, but if `-s` were the only way to express that, it would be better to include `<SOLUTION_FOLDER>`. Since consistency is good, I would say that if there is _one_ case in which we can't have a well-named switch, we should do it across the board. Otherwise, it's fine to take it out.

I'll take it out for now.

Reply:
To me it makes sense to show the argument in the synopsis in all cases.  Otherwise you can't tell the difference between a switch like `--in-root` which doesn't accept an argument and one like `--solution-folder` which requires one.